launch/nav.launch

```diff
@@ -0,0 +1,72 @@
+<launch>
+  <arg name="model" default="$(find mrover)/rover_description/rover.urdf" />
+  <arg name="gui" default="false" />
+  <param name="robot_description" command="xacro $(find mrover)/rover_description/rover.xacro" />
+  <arg name="rvizconfig" default="$(find mrover)/nav.rviz" />
+
+  <node if="$(arg gui)" name="joint_state_publisher" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui" />
+  <node unless="$(arg gui)" name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
+  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
+  <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />
+
+  <include file="$(find gazebo_ros)/launch/empty_world.launch">
+    <env name="GAZEBO_MODEL_PATH" value="$(find mrover)/env_description" />
+    <arg name="world_name" value="$(find mrover)/env_description/world.world" />
+    <arg name="paused" value="false" />
+    <arg name="use_sim_time" value="true" />
+    <arg name="gui" value="true" />
+    <arg name="headless" value="false" />
+    <arg name="debug" value="false" />
+    <arg name="verbose" default="false" />
+    <arg name="gui_required" value="true" />
+  </include>
+
+  <node name="spawn_model" pkg="gazebo_ros" type="spawn_model" output="screen" args="
+    -urdf -param robot_description
+    -model rover" />
+
+  <!-- Perception-->
+  <node name="aruco_detect" pkg="mrover" type="aruco_detect">
+    <param name="publish_images" value="true" />
+    <param name="fiducial_len" value="0.20" />
+    <param name="dictionary" value="0" /> <!-- DICT_4X4_50 -->
+    <remap from="camera" to="camera/color/image_raw" />
+    <remap from="camera_info" to="camera/color/camera_info" />
+  </node>
+
+  <!-- Navigation -->
+  <!-- This publishes the transforms of the course waypoints -->
+  <node name="debug_course_publisher" pkg="mrover" type="debug_course_publisher.py" />
+
+  <!--
+    ============
+    Localization
+    ============
+    Frames in the free tree:
+    odom: The topmost global frame
```

Comment:
This is a really confusing name for this frame. Odometry would imply the pose estimate of the rover - at least in my head. Should we call it `map` or `world`. I'm in favor of renaming the Gazebo frame to `ground_truth` too if possible so its unambigiously clear which side of the tree is okay to use vs not for non-sim autonomy code.

Reply:
Yes I agree, ideally we conform to: https://www.ros.org/reps/rep-0105.html

Reply:
interesting, so I suppose the correct frame here would be the map frame. Then the odom frame can be something else. Right now we don't have a way of localizing in an odom frame persay, but maybe with a percep-based motion model thats something we can have, and likely something we need and want for gate. @rbridges12

Reply:
yeah agreed, this might be a good way to balance local vs global accuracy, will have to do testing to see which sensors/motion models would be best for integrating into the odom frame